main.go

```diff
@@ -381,20 +421,50 @@ func runCLIApp(c *cli.Context) (err error) {
 	{
 		mfs, err = mountWithArgs(bucketName, mountPoint, flags, mountConfig)
 
-		if err == nil {
+		// This utility is to absorb the error
+		// returned by daemonize.SignalOutcome calls by simply
+		// logging them as error logs.
+		callDaemonizeSignalOutcome := func(err error) {
+			if err2 := daemonize.SignalOutcome(err); err2 != nil {
+				logger.Errorf("Failed to signal to daemon: %v", err2)
```

Comment:
The correct message should be: "Failed to signal via daemon to parent process" 
`daemonize.SignalOutcome(err)` is called via daemon process.

Reply:
fixed.

Reply:
fixed.